pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1254,7 +1254,13 @@ self: super: {
   });
 
   # The LTS-14.x version of their dependencies are too old.
-  cabal-plan = super.cabal-plan.overrideScope (self: super: { optparse-applicative = self.optparse-applicative_0_15_1_0; ansi-terminal = self.ansi-terminal_0_10_2; base-compat = self.base-compat_0_11_1; semialign = self.semialign_1_1; time-compat = doJailbreak super.time-compat; });
+  cabal-plan = super.cabal-plan.overrideScope (self: super: {
+    optparse-applicative = self.optparse-applicative_0_15_1_0;
+    ansi-terminal = self.ansi-terminal_0_10_3;
+    base-compat = self.base-compat_0_11_1;
+    semialign = self.semialign_1_1;
+    time-compat = doJailbreak super.time-compat;
+  });
```

Comment:
I formatted this to be on multiple lines to make it a little easier to read.

The only change I made is to change `self.ansi-terminal_0_10_2` to `self.ansi-terminal_0_10_3`.